examples/cli/build.md

```diff
@@ -59,7 +59,7 @@ The default `Sequence` for CPU and GPU benchmarking performs the following build
 1. Use ONNX ML Tools to the convert the optimized ONNX file to float16
 1. Set the `build_status=successful_build` property
 
-You can see this if you already did the [Hello World tutorial](#hello-world) by running the command:
+You can see this if you already did the [Hello World tutorial](https://github.com/groq/mlagility/blob/main/examples/cli/readme.md#hello-world) by running the command:
```

Comment:
Line48 Missed Deadlink
> See the [Build Only documentation](https://github.com/groq/mlagility/blob/main/tools_user_guide.md#build-only) for more details.

Reply:
There's a couple of this file, so maybe search for the guide and replace all the links to it? 
tools_user_guide.md

Reply:
I think it's just Line 48 and 52 that have the deadlink

Reply:
got them, thanks! did a repo-wide search just to be sure that was all of them.